plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -230,6 +230,7 @@ private QueryRunner createDeltaLakeQueryRunner()
                         .put("hive.metastore-cache-ttl", TEST_METADATA_CACHE_TTL_SECONDS + "s")
                         .put("delta.register-table-procedure.enabled", "true")
                         .put("hive.metastore.thrift.client.read-timeout", "1m") // read timed out sometimes happens with the default timeout
+                        .put("delta.vacuum.logging.enabled", "true")
```

Comment:
Is there a test when we VACUUM and logging is disabled? If not add one.

Reply:
Added sessionProperty for vacuum_logging_enabled and improved the existing test